images/macos/provision/core/aws.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash -e -o pipefail
 
+source ~/utils/utils.sh
+
 echo Installing aws...
-curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
+AWSCLIURL="https://awscli.amazonaws.com/AWSCLIV2.pkg"
+download_with_retries $AWSCLIURL "/tmp"
+pushd /tmp
```

Comment:
Can we just do `sudo installer -pkg /tmp/AWSCLIV2.pkg -target /` and get rid of pushd and popd?

Reply:
And I'm not sure that we prune /tmp at the end of the build, so I'd suggest clean it up manually

Reply:
Done